### ISSUE TITLE
File.Move Issue

### ISSUE THREAD

**Issue by wgv-mrunnels** [jordanlytle/SmbAbstraction#24]
File.Move appears to just be doing a copy leaving the original file in place. Does not match expected behavior of System.IO.File.Move.

```
internal void Move(string sourceFileName, string destFileName, ISMBCredential sourceCredential, ISMBCredential destinationCredential)
{
    using (Stream sourceStream = OpenRead(sourceFileName, sourceCredential))
    {
          using (Stream destStream = OpenWrite(destFileName, destinationCredential))
          {
               sourceStream.CopyTo(destStream, Convert.ToInt32(_maxBufferSize));
          }
    }
}
```

**Comment by Jo0** [comment]
Got changes and tests staged. Will be publishing a new package soon.

**Comment by Jo0** [comment]
Just tested published package version 1.1.6

Move now remove file specified in `sourceFileName`
